ctms/auth.py

```diff
@@ -133,7 +133,12 @@ def __init__(
 
     async def __call__(self, request: Request) -> Optional[str]:
         authorization: str = request.headers.get("Authorization")
-        scheme, param = get_authorization_scheme_param(authorization)
+
+        # TODO: Try combining these lines after FastAPI 0.61.2 / mypy update
+        scheme_param = get_authorization_scheme_param(authorization)
+        scheme: str = scheme_param[0]
+        param: str = scheme_param[1]
```

Comment:
I didn't understand why this was necessary. FastAPI defines [get_authorization_scheme_param](https://github.com/tiangolo/fastapi/blob/22528373bba6a654323de416ad5c867cbadb81bb/fastapi/security/utils.py#L4-L8) as returning ``Tuple[str, str]``, but mypy doesn't recognize the two-item assignment as ``str``. It could be a FastAPI packaging issue (a ``py.typed`` file may be missing), or a mypy issue (FastAPI pins the mypy version). I left a note to see if the one-line version will be valid in the future.